Config.cmake/runtime_test.cmake

```diff
@@ -171,21 +168,35 @@ add_test(ctest_fmi_zip_unzip_test fmi_zip_unzip_test)
 add_test(ctest_fmi_zip_zip_test fmi_zip_zip_test)
 
 
+# ------------------------------------------------------------------------------
+# Catch2
+# ------------------------------------------------------------------------------
+
+set(CMAKE_CXX_STANDARD 11) # Required by Catch2
+add_library(Catch INTERFACE)
+
+# Catch2 consists of only headers, but we still want to compile its main function
+# in a separate object file so we don't need to do it for each test.
+add_library(catch2_main STATIC ${FMIL_TEST_DIR}/catch2_main.cpp)
+target_link_libraries(catch2_main Catch)
```

Comment:
I decided to just push this in now since it was getting unbearably slow.

https://github.com/catchorg/Catch2/blob/v2.x/docs/slow-compiles.md

That's actually how we do it for our other code that use Catch2 as well.